src/Db.php

```diff
@@ -17,8 +17,10 @@ public static function getInstance(): Db {
   private function __construct() {
     $this->config = parse_ini_file($this->settings_file);
     $options = array(
-      'idle_timeout_micros' => 200000,
+      'idle_timeout_micros' => 2000000,
       'expiration_policy' => 'IdleTime',
+      'per_key_connection_limit' => 20,
+      'pool_connection_limit' => 20
```

Comment:
> `pool_connection_limit' => 20`

**Can you explain the justification for lowering this from the default value?**

The default is `50` connections.  _It should be noted that this value is the total number of pool connections, regardless of the MySQL server hostname, port, database, and username._

In theory, this value would be irrelevant, as the `per_key_connection_limit` would be hit when using a single database server.  However, it is possible to use multiple databases.  In fact, PR #419 provides the ability to use database replication across numerous replicated databases (though, to be honest, in its current form #419 is highly unlikely to ever be merged).  Regardless, a user could utilize multiple databases which would bring this value into relevancy.

Please see the comment on the line above (`per_key_connection_limit' => 20`) for more details and thoughts on this change.

Reply:
I answered the other comment, I agree that this is irrelevant if we have the other line.